lib/logs/docker-compose.logs.yaml

```diff
@@ -6,12 +6,13 @@ services:
       - GF_PATHS_PROVISIONING=/etc/grafana/provisioning
       - GF_SERVER_ROOT_URL=${BENTO_PRIVATE_GRAFANA_URL}
       - GF_SERVER_SERVE_FROM_SUB_PATH=true
-      - GF_SECURITY_COOKIE_SAMESITE=strict
+      - GF_SECURITY_COOKIE_SAMESITE=none
       - GF_SECURITY_COOKIE_SECURE=true
       - GF_SECURITY_ALLOW_EMBEDDING=true
```

Comment:
does it still need to allow embedding if we no longer do the iframe stuff?

Reply:
no, removed it